web/src/compositions/useWPTitle.ts

```diff
@@ -0,0 +1,7 @@
+import { useTitle } from "@vueuse/core"
+import type { Ref } from "vue";
+import { computed } from "vue"
+
+export function useWPTitle(elements: Ref<string[]>) {
```

Comment:
Might be more readable to use sth like: 


```suggestion
export function useWPTitle(...elements: Ref<string>) {
```